### ISSUE TITLE
add "finetune-pred" feature, label.update_finetune method

### ISSUE THREAD

**Issue by nadahlberg** [freelawproject/classifier-experiments#65]
The `update_finetune` method will assemble a training dataset using label.get_trainset and finetune a model for a particular label. Then it will push predictions to the DB. The 'finetune-pred' feature will represent a positive prediction from the latest finetune.
